src/dcqc/tests/tests.py

```diff
@@ -158,3 +158,27 @@ def generate_process(self) -> Process:
             command_args=command_args,
         )
         return process
+
+
+class TiffTag306DateTimeTest(ExternalTestMixin, TestABC):
+    tier = 4
+
+    def generate_process(self) -> Process:
+        file = self.get_file()
+        path = file.local_path.as_posix()
+        command_args = [
+            "!",  # negate exit status
```

Comment:
So for whatever reason, this `!` is getting passed to the command that is executed in Nextflow as '!'. Resulting in an error. Example (from process `.command.sh`):
```
#!/bin/bash -euo pipefail
( ('!grep' -E -i -a -q ''"'"'date|time'"'"'' /Users/bmacdonald/Development/repos/nf-dcqc/work/83/35c8f24d1450580c75cff5f7e9d94c/dcqc-staged-tsdnarcr/single-channel.tif) > "std_out.txt" 2> "std_err.txt"; echo $? > "exit_code.txt" ) || true

```